lib/wrapbox/runner/docker.rb

```diff
@@ -54,7 +56,23 @@ def run_cmd(cmds,  container_definition_overrides: {}, environments: [])
             )
           end
         end
-        ths.each(&:join)
+        ths.each { |th| th&.join }
+
+        true
+      rescue SignalException => e
+        sig = e.is_a?(Interrupt) ? "SIGINT" : e.signm
```

Comment:
How about this?

```rb
sig = "SIG#{Signal.signame(e.signo)}"
```

Reply:
I misunderstood about it, because`Interrupt#signm` is blank. 
I assumed that `e.signm` does not return value.
Thanks.